qutip/solver/flimesolve.py

```diff
@@ -432,25 +425,25 @@ def __init__(
                 self.Nt = 2**4
         self.time_sense = time_sense
 
-        RateDic = _floquet_rate_matrix(
+        rateDict = _floquet_rate_matrix(
             floquet_basis,
             self.Nt,
-            self.floquet_basis.T,
             c_ops,
             c_op_rates,
-            Hargs,
-            time_sense=time_sense)
-        Rate_Qobj_list = [Qobj(
-            RateMat, dims=[self.floquet_basis.U(0).dims, self.floquet_basis.U(0).dims], type="super", superrep="super", copy=False
-        ) for RateMat in RateDic.values()]
-        R0 = Rate_Qobj_list[0]
-
-        Rt_timedep_pairs = [list(
-            [Rate_Qobj_list[idx], 'exp(1j*' + str(list(
-                RateDic.keys())[idx]
-                * list(Hargs.values())[0]) + '*t)'])
-            for idx in range(0, len(Rate_Qobj_list))]
-        Rate_matrix_timedep_list = [R0, *Rt_timedep_pairs[1::]]
+            time_sense=time_sense
+        )
+
+        R0 = rateDict[0]
+
+        omega = 2 * np.pi / floquet_basis.T
+
+        Rt_timedep_pairs = [
+            [rate_matrix, _Phase(omega * key)]
```

Comment:
Many users don't have cython installed, it's better to default to a function call for them.

Reply:
I'll fix this